.env

```diff
@@ -0,0 +1,3 @@
+SPOTIFY_ID=
```

Comment:
Instead of pushing `.env` file, better to follow the convention/standards and push `.env.example` with some dummy env values and add `.env` file to `.gitignore`